MANIFEST.in

```diff
@@ -5,7 +5,9 @@ include LICENSE
 include README.rst
 
 recursive-include tests *
+graft api
```

Comment:
This doesn't look right, there's no `api` directory in the root directory.

https://docs.python.org/3/distutils/commandref.html?highlight=graft#creating-a-source-distribution-the-sdist-command

Reply:
Sorry, this was a test trying to include the `alfasim_sdk_api` (that I renamed to API for test purposes) from the root dir in the source-distribution that didn't succeed.